README.md

```diff
@@ -136,7 +136,7 @@ npm run lint:all
 | Option                    | Description                                                                                                                                                                                                                                                      |
 | ------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `--browsers` or `-b`      | You can specific browser(s) to run the unit test. List of available browsers are following: `chrome`, `firefox` and `ie` <br>You can check the list of available browsers by add option `--help`.  <br>Usage: `npm run test button -- --browsers chrome firefox` |
-| `--browserstack` or `-bs` | You can run test on BrowserStack with specific browser(s). List of available browsers are following: `chrome`, `firefox`, `edge_previous`, and aliases `default`, `supported`, and etc. <br>This command option required BrowserStack API Key you must setting the three environment variables `BROWSERSTACK_USERNAME`, `BROWSERSTACK_ACCESS_KEY`, and `BROWSERSTACK_BUILD`. You can create API key from [here](https://www.browserstack.com/accounts/settings) <br>Usage: `npm run test button -- --browserstack chrome firefox_previous`
+| `--browserstack` or `-bs` | You can run test on BrowserStack with specific browser(s). List of available browsers are following: `chrome`, `firefox`, `edge_previous`, and aliases `default`, `supported`, and etc. <br>This command option required BrowserStack API Key you must setting the three environment variables `BROWSERSTACK_USERNAME`, `BROWSERSTACK_ACCESS_KEY`, and `BROWSERSTACK_BUILD`. You can create API key from [here](https://www.browserstack.com/accounts/profile) <br>Usage: `npm run test button -- --browserstack chrome firefox_previous`
```

Comment:
Please explain BROWSERSTACK_BUILD as we discussed earlier. I wasn't sure what to put value to this env until I ask you. I think many people will wonder the same thing.